apps/webapp/app/routes/resources.orgs.$organizationSlug.select-plan.tsx

```diff
@@ -589,21 +844,28 @@ function PricingHeader({
 function TierLimit({ children, href }: { children: React.ReactNode; href?: string }) {
```

Comment:
I added a tooltip when you hover over the "$x usage included" link so explain where it takes you